test/unit/view.js

```diff
@@ -1,317 +1,136 @@
       result.should.eql(data)
       done()
     })
   })
 
-  it('should return html when calling `render()`', function (done) {
-    var name = 'test'
+  it("should return html when calling `render()`", function(done) {
+    var name = "test"
     var schema = TestHelper.getPageSchema()
-    schema.template = 'test.dust'
+    schema.template = "test.dust"
 
     // load a template
-    var template = '{#names}{title} {name}{~n}{/names}'
-    var compiled = dust.compile(template, 'test', true)
+    var template = "{#names}{title} {name}{~n}{/names}"
+    var compiled = dust.compile(template, "test", true)
     dust.loadSource(compiled)
 
-    var req = { url: '/test' }
+    var req = { url: "/test" }
     var p = page(name, schema)
     var v = view(req.url, p, false)
 
-    var data = { 'title': 'Sir', 'names': [{ 'name': 'Moe' }, { 'name': 'Larry' }, { 'name': 'Curly' }] }
-
-    v.setData(data)
-    v.render(function (err, result) {
-
-      var expected = 'Sir Moe\nSir Larry\nSir Curly\n'
-      result.should.eql(expected)
-      done()
-    })
-  })
-
-  it.skip('should throw an error if the configured helper path cannot be found', function (done) {
-    // set a helper path in config
-    var pathConfig = {
-      paths: {
-        helpers: 'test/app/utils/not/here/helpers'
-      }
+    var data = {
+      title: "Sir",
+      names: [{ name: "Moe" }, { name: "Larry" }, { name: "Curly" }]
     }
 
-    TestHelper.updateConfig(pathConfig).then(() => {
-      var name = 'test'
-      var schema = TestHelper.getPageSchema()
-      schema.template = 'test.dust'
-
-    // load a template
-      var template = '<h1>Code Helper Example</h1>'
-      template += '{@code}'
-      template += "alert('Hello World')"
-      template += '{/code}'
-
-      var compiled = dust.compile(template, 'test', true)
-      dust.loadSource(compiled)
-
-      var req = { url: '/test' }
-      var p = page(name, schema)
-
-      should.throws(function () { view(req.url, p, false) }, Error)
-
-      done()
-    })
-  })
-
-  it('should still render if custom dust helper cannot be found when calling `render()`', function (done) {
-    var name = 'test'
-    var schema = TestHelper.getPageSchema()
-    schema.template = 'test.dust'
-
-    // load a template
-    var template = '<h1>Code Helper Example</h1>'
-    template += '{@code}'
-    template += "alert('Hello World')"
-    template += '{/code}'
-
-    var expected = '<h1>Code Helper Example</h1>'
-
-    var compiled = dust.compile(template, 'test', true)
-    dust.loadSource(compiled)
-
-    var req = { url: '/test' }
-    var p = page(name, schema)
-    var v = view(req.url, p, false)
-
-    v.render(function (err, result) {
+    v.setData(data)
+    v.render(function(err, result) {
+      var expected = "Sir Moe\nSir Larry\nSir Curly\n"
       result.should.eql(expected)
       done()
     })
   })
-
-  it('should have access to custom dust helpers when calling `render()`', function (done) {
```

Comment:
The tests removed from this file have been added to the [Dust interface module](https://github.com/dadi/web-dustjs).